### PR TITLE
Add company and back button to forms for sponsors

### PR DESCRIPTION
**Pull request by florianschmidt1994**
@dominikglandorf Can you review this?

### CODE REVIEW

File: src/client/react/Components/Sponsor/CompanyInformationForm.jsx [hunk 1]
```diff
@@ -105,6 +109,7 @@ export default class CompanyInformationForm extends React.Component {
               showErrorList={false}
               formData={this.state.formData}
               onSubmit={this.props.onSubmit}>
+          <Button className="primary" type="button" onClick={this.props.onBack}>{t('back')}</Button>
```
Comment (dominikglandorf) [inline src/client/react/Components/Sponsor/CompanyInformationForm.jsx:112]: Where is onBack defined?

Comment (florianschmidt1994) [reply src/client/react/Components/Sponsor/CompanyInformationForm.jsx:112]: In the parent component

Comment (dominikglandorf) [reply src/client/react/Components/Sponsor/CompanyInformationForm.jsx:112]: Works for me!

**Review by dominikglandorf — APPROVED** [review]
Really great!

**Pull request merged**